DEV-9/FileReader.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace ReplacementOfCharacterGroups
+{
+  public class FileReader
```

Comment:
bad name, this is   the specific file reader, has to be obsequious from the  class name